### PR TITLE
complete the translation of management-tools.md

### PR DESCRIPTION
**Pull request by zRich**

### CODE REVIEW

**Comment by zRich** [comment]
review required!

@lfapac-open-source-evangelist/review

File: content/zh-CN/guides/management-tools.md [hunk 1]
```diff
@@ -13,35 +13,20 @@ title: 管理开源项目的工具
...
-  - [For Contributor License Agreements](#for-contributor-license-agreements)
   - [对于贡献者许可协议](#对于贡献者许可协议)
-  - [GitHub Management at Corporate Scale](#github-management-at-corporate-scale)
   - [GitHub 在企业规模上的管理](#github-在企业规模上的管理)
-  - [Project Quality](#project-quality)
   - [项目质量](#项目质量)
-- [Tools for communications and collaboration](#tools-for-communications-and-collaboration)
 - [通信与协作工具](#通信与协作工具)
-- [Tools for corporate-scale GitHub management](#tools-for-corporate-scale-github-management)
-- [企业规模 GitHub 管理工具](#企业规模-github-管理工具)
```
Comment (windsonsea) [inline content/zh-CN/guides/management-tools.md:43]: Is this zh line (ilne 43) intentional to remove?

File: content/zh-CN/guides/management-tools.md [hunk 2]
```diff
@@ -13,35 +13,20 @@ title: 管理开源项目的工具
...
-- [Final words](#final-words)
+- [企业级 GitHub 管理工具](#企业级-github-管理工具)
 - [最后的话](#最后的话)
 - [Acknowledgements](#acknowledgements)
```
Comment (windsonsea) [inline content/zh-CN/guides/management-tools.md:31]: This en line can be removed too

File: content/zh-CN/guides/management-tools.md (hunk 1 above)
Comment (node) [reply content/zh-CN/guides/management-tools.md:43]: > Is this zh line (ilne 43) intentional to remove?

We'd better focus on the right side file when review , right ? :smile:

File: content/zh-CN/guides/management-tools.md [hunk 3]
```diff
@@ -13,35 +13,20 @@ title: 管理开源项目的工具
...
-  - [Archiving and release management](#archiving-and-release-management)
   - [归档和发布管理](#归档和发布管理)
-- [Tools for tracking project health](#tools-for-tracking-project-health)
 - [项目健康跟踪工具](#项目健康跟踪工具)
-  - [For better code reviews:](#for-better-code-reviews)
   - [为了更好的代码审查：](#为了更好的代码审查)
-  - [For Contributor License Agreements](#for-contributor-license-agreements)
   - [对于贡献者许可协议](#对于贡献者许可协议)
-  - [GitHub Management at Corporate Scale](#github-management-at-corporate-scale)
   - [GitHub 在企业规模上的管理](#github-在企业规模上的管理)
```
Comment (node) [inline content/zh-CN/guides/management-tools.md:26]: 企业级 GitHub 管理？

File: content/zh-CN/guides/management-tools.md [hunk 4]
```diff
@@ -318,153 +193,84 @@ The TODO Group also offers a [helpful list that adds other tools](https://GitHub
...
-
-Communications can also be fostered through social media platforms, web portals, open source project repositories and other places where input, questions and discussions can be found and fostered.
-
-Then there’s [Slack](https://slack.com/), which is an online team project management and communications platform where users can access and share messages and files, organize workflows, perform searches for information and more. Slack can be configured to receive notifications for support requests, code check-ins, error logs and other tasks as well.
-
-And don’t forget your company’s public relations and marketing staff when it comes to shouting out your company’s participation and support of open source. Social media accounts with sites including Twitter, Reddit, Facebook, LinkedIn and others are important, as well as the use of internal and external blogs and websites. Customer Relationship Management (CRM) software, as well as email blasts and newsletters, can help companies keep customers and clients informed about their open source progress.
+* [CodeClimate](https://codeclimate.com/) – Code Climate 让组织能够通过在开发工作流程中完全可配置的测试覆盖率和可维护性数据来控制他们的代码质量。对于开源项目，它是免费的！
 
 ## 通信与协作工具
```
Comment (node) [inline content/zh-CN/guides/management-tools.md:235]: 沟通与协作工具

File: content/zh-CN/guides/management-tools.md [hunk 5]
```diff
@@ -318,153 +193,84 @@ The TODO Group also offers a [helpful list that adds other tools](https://GitHub
...
-The creation of an open source program office and the selection of a package of critical tools to get your efforts started are within your grasp. And they are likely already inspiring great anticipation among your developers, many of whom are probably already contributing to open source projects on their own (or at work, under cover of darkness).
+微软创建的工具之一是一个定制的自助 [GitHub 管理和入职门户](http://www.jeff.wilcox.name/2015/11/azure-on-github/)，用于组织其项目、仓库和团队。在最简单的层面上，这个基于网络的门户允许开发人员将他们的微软公司 ID 映射到他们的 GitHub ID，从而增强系统安全性，并有助于简化大量重要项目中涉及的开发人员的组织工作。
 
-By collaborating on open source projects and inviting others to collaborate with you, your company can gain immeasurable benefits and drive its progress forward with energy and innovation.
+该门户还允许员工通过 GitHub 和微软进行身份验证，从而创建他们身份的“虚拟链接”，使他们能够在工作中获得所需权限，具体取决于他们的工作角色。如果员工离开公司，该系统可以进行调整，以根据需要删除或重新分类他们的访问权限。
 
-Having the right tools is critical to empowering your company’s open innovation.
+该门户运行在一个或多个云服务器上，并依靠缓存来帮助会话管理并减轻 GitHub API 的压力。作为其不断增长的开源努力的一部分，微软门户每天平均有大约 1000 名独特用户使用这个工具，这些努力现在包括了超过 10000 名使用、贡献和发布开源代码的工程师。
 
 ## 最后的话
```
Comment (node) [inline content/zh-CN/guides/management-tools.md:265]: 结束语

File: content/zh-CN/guides/management-tools.md [hunk 6]
```diff
@@ -318,153 +193,84 @@ The TODO Group also offers a [helpful list that adds other tools](https://GitHub
...
-建立一个开源项目办公室，并选择一套关键工具来启动你的努力，这些都在你的掌握之中。它们很可能已经在你的开发人员中间引发了极大的期待，其中许多人可能已经在自己的项目中（或在工作中，在黑暗中）为开源项目做出了贡献。
+创建一个开源项目办公室并选择一套关键工具来启动你的工作，这是你力所能及的事情。而且它们很可能已经激发了你开发人员的极大期待，其中许多人可能已经在自行（或在工作中秘密地）为开源项目做出贡献。
 
-通过在开源项目上合作，并邀请其他人与你合作，你的公司可以获得无法估量的好处，并以活力和创新推动其进步。
+通过在开源项目上进行协作并邀请其他人与你合作，你的公司可以获得无可估量的利益，并通过充满活力和创新推动其进步。
 
-拥有合适的工具对于赋予你公司开放式创新的能力至关重要。
+拥有合适的工具对推动公司开放创新至关重要。
 
 ## Acknowledgements
```
Comment (node) [inline content/zh-CN/guides/management-tools.md:275]: 英文需删除

File: content/zh-CN/guides/management-tools.md [hunk 7]
```diff
@@ -475,7 +281,7 @@ The affiliations of the contributors are from when the article was originally pu
 
 ## 致谢
 
-贡献者的所属单位是根据文章最初发布时的情况：
+贡献者的所属单位是根据文章2019年最初发布时的情况：
```
Comment (node) [inline content/zh-CN/guides/management-tools.md:284]: 贡献者所属组织以文章于2019年最初发布时为准。

**Review by node — CHANGES_REQUESTED** [review]

File: content/zh-CN/guides/management-tools.md [hunk 8]
```diff
@@ -166,18 +121,6 @@ Your applications, for example, could include several thousand open source compo
 
 >“您必须了解您的风险概况，因为最终扫描的一切都是关于风险管理。您可以把头埋在沙子里，然后只是相信并希望自己没事。或者您可以说‘如果我被起诉，那将毁了我的生意。’您需要真正确定。所以，您打开软件包，然后查看所有代码行，并找到可能存在的所有内容。” – Jeff McAffer，微软开源项目办公室主任。
```
Comment (HudsonShi) [inline content/zh-CN/guides/management-tools.md:122]: 您需要真正确定（您的风险概况）

Comment (HudsonShi) [reply content/zh-CN/guides/management-tools.md:122]: 并找到可能存在（风险）的所有内容

**Review by HudsonShi — COMMENTED** [review]
Seems good to me but need a little modification.

**Comment by zRich** [comment]
@windsonsea review is required from you

**Review by node — APPROVED** [review]
LGTM

**Pull request merged**